### DISCUSSION TITLE
Lab4-1 synthesis error

### DISCUSSION THREAD

**Discussion by M11107004** [bol-edu/HLS-SOC-Discussions#136] [Q&A]
拿counter.v跟bram.v做合成發生容量不夠的問題，導致合成失敗

![E0421CB9-7B25-465C-B9A6-2E24652047EF](https://github.com/bol-edu/HLS-SOC-Discussions/assets/132893589/ef51af71-518c-499f-b48e-c88484d77e02)

這樣的結果是對的嗎？或是要如何解決

![5AD1D1DA-6A78-4390-B185-36C177A21FD9](https://github.com/bol-edu/HLS-SOC-Discussions/assets/132893589/37828932-f023-48d6-8a6a-4da0cba366f7)
![95A77E3B-DD09-45A3-8FAB-FE2A83AA75BC](https://github.com/bol-edu/HLS-SOC-Discussions/assets/132893589/6abeb08f-2d61-40db-810c-135305552d20)

**Comment by MODKWODK** [comment]
或許可以試一下這個方法

https://medium.com/@ihsanalhafiz28/how-to-solve-vivado-crash-on-ubuntu-while-running-4b445d609fa5
